orientation/FND_17_SQL.md

```diff
@@ -0,0 +1,159 @@
+# Introduction to SQL
+
+### Installation of SQLite (if needed)
+
+To get started, type the following command to check if you already have SQLite installed.
+
+```bash
+$ sqlite3
+```
+
+And you should see:
+
+```
+SQLite version 3.7.15.2 2014-08-15 11:53:05
+Enter ".help" for instructions
+Enter SQL statements terminated with a ";"
+sqlite>
+```
+
+If you do not see above result, then it means you do not have SQLite installed on your machine. Follow the appropriate instructions below.
+
+#### For Windows
+
+Go to [SQLite Download page](http://www.sqlite.org/download.html) and download the precompiled binaries for your machine. You will need to download `sqlite-shell-win32-*.zip` and `sqlite-dll-win32-*.zip` zipped files.
+
+Create a folder `C:\sqlite` and unzip the files in this folder which will give you `sqlite3.def`, `sqlite3.dll` and `sqlite3.exe` files.
+
+Add `C:\sqlite` to your [PATH environment variable](http://dustindavis.me/update-windows-path-without-rebooting/) and finally go to the command prompt and issue `sqlite3` command.
+
+#### For Mac
```

Comment:
Let's get an 'official' Linux install instruction in here, too. We won't be making it mandatory in the future, but there's always someone running it, regardless